strax/processing/general.py

```diff
@@ -135,13 +137,20 @@ def _fc_in(a_starts, b_starts, a_ends, b_ends, result):
 
 @export
 def split_by_containment(things, containers):
-    """Return list of thing-arrays contained in each container
+    """
+    Return list of thing-arrays contained in each container. Result is
+    returned as a numba.typed.List or list if containers are empty.
 
-    Assumes everything is sorted, and containers are nonoverlapping
+    Assumes everything is sorted, and containers are non-overlapping.
     """
     if not len(containers):
         return []
```

Comment:
Should this be a typed empy list?

Reply:
Yes, thanks for spotting. It took a small hack to define the list type.